src/main/java/com/jmb/springfactory/service/task/TaskValidatorService.java

```diff
@@ -57,8 +57,8 @@ public void validateOnUpdate(Object obj) {
     generalValidate(taskDto);
 
     serviceLog.info("Validating if create user has not changed");
-    if (creatorHasChanged.test(taskDto)) 
-      throw new ValidationException("The task creator cant be changed");
+    //if (creatorHasChanged.test(taskDto)) 
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)